modules/query/src/test/scala/ch/epfl/bluebrain/nexus/admin/query/builder/FilteredQuerySpec.scala

```diff
@@ -37,6 +40,8 @@ class FilteredQuerySpec extends WordSpecLike with Matchers with Resources with E
   private val hasRead: HasReadProjects =
     applyRef[HasReadProjects](FullAccessControlList(
       (Identity.Anonymous(), Path./, Permissions(Read, Permission("projects/read"))))).toOption.get
+  implicit val idRef: IdResolvable[ProjectReference] = (a: ProjectReference) =>
+    IdRef(applyRef[Prefix]("projects").toOption.get, applyRef[Namespace]("https://localhost/project").toOption.get, a)
```

Comment:
Since it is compile time, you can simply use `"projects": Prefix` and `"https://localhost/project": Namespace` as you can see [here](https://github.com/BlueBrain/nexus-admin/blob/bd3c5a09288e55205af566b74a2f015f87f44282/modules/refined/src/test/scala/ch/epfl/bluebrain/nexus/admin/refined/IdSpec.scala#L43)